lib/controller.js

```diff
@@ -941,6 +943,8 @@ controller.updateTesselWithVersion = function(opts, tessel, currentVersion, buil
 controller.tesselEnvVersions = opts => {
   opts.authorized = true;
   return controller.standardTesselCommand(opts, tessel => {
+    var pkg = path.join(__dirname, '..', 'package.json');
+    var cliVersion = fs.readJsonSync(pkg).version;
```

Comment:
I did have this at first, then I was updating the tests, and I wasn't sure how to mock that. If that `require` isn't mocked then every time a new version of the cli was released, we would have to update the tests.

Thoughts?